core/src/telemetry/metrics.rs

```diff
@@ -327,6 +335,7 @@ impl Instruments {
             sticky_cache_hit: meter.counter("sticky_cache_hit"),
             sticky_cache_miss: meter.counter("sticky_cache_miss"),
             sticky_cache_size: meter.histogram(STICKY_CACHE_SIZE_NAME),
+            sticky_cache_evictions: meter.counter("sticky_cache_total_forced_eviction"),
```

Comment:
I saw this in the metrics doc and could figure out what "forced" is in this context, looks like here it's just "normal" eviction.

https://docs.temporal.io/references/sdk-metrics#sticky_cache_total_forced_eviction

Reply:
Yeah, it's a kinda bad name but can't really change it just for this